pkg/phlaredb/block_querier_test.go

```diff
@@ -175,3 +177,148 @@ func TestSelectMatchingProfilesCleanUp(t *testing.T) {
 	})
 	require.Error(t, err)
 }
+
+func TestSeries(t *testing.T) {
```

Comment:
The test case for getting series labels from a single block. I still need to try create an e2e test case.